secrets.yml

```diff
@@ -0,0 +1,2 @@
+CF_API_ENDPOINT: https://api.sys.pcf.itd.conjur.net
```

Comment:
Is it okay to put this in the repo or should it also be summon-ed?

Reply:
Well... I can definitely see where you're coming from. Although that cat is also already out of the bag in the service broker repo.

My gut says we should probably file a ticket that includes changing the domain for the cluster and updating both repos to pull it from ConjurOps.

What do you think?

Reply:
Sounds good, but I think we might as well get it into conjurops now and use the reference here since we haven't merged to master. It's pretty low efforts and will reduce the number of things we need to touch when that ticket gets fulfilled.